### PR TITLE
Fixes to foodcalc and changes to Namekians and Nam Genome Biodroids.

### PR DESCRIPTION
**Pull request by PetalFoxx**
Summary:

1. Made Nameks actually have 'hunger', but they will not get the penalties associated
2. Changed the benefit to Bio-Androids that had the Namekian gene have Stretchy arms instead as it's more impactful
3. Bugfix: Stretchy Arm logic required an elevation difference of two, rather than one.
4. Bugfix: Extraneous logic in the food xp calc meant you could get no xp post level 100

### CODE REVIEW

**Comment by PetalFoxx** [comment]
5. Also added the ability to hopefully choose weight and height at gen. Needs testing, chargen is implemented oddly and probably needs reworking.

**Comment by PetalFoxx** [comment]
Please triple check the combat.cpp changes, I've added in the suppression logic, it should work.

**Review by volundmush — COMMENTED** [review]
Need to increase NUM_CON_TYPES and ensure it's mirrored in constants.cpp's array if this gets edited. It looks like CON_WEIGHT was inserted. I"ll take care of this this time, just be aware for next time. :)

File: include/dbat/defs.h [hunk 1]
```diff
@@ -969,20 +969,21 @@
...
+#define CON_POBJ         69
+#define CON_ALIGN        70
+#define CON_SKILLS       71
+#define CON_USER_TITLE   72
+#define CON_GENOME       73
+#define CON_COPYOVER     74
+#define CON_LOGIN        75
+#define CON_QUITGAME     76
 
 #define NUM_CON_TYPES 76
```
Comment (volundmush) [inline include/dbat/defs.h:988]: Need to increase NUM_CON_TYPES and ensure it's mirrored in constants.cpp's array if this gets edited. It looks like CON_WEIGHT was inserted. I"ll take care of this this time, just be aware for next time. :)

**Pull request merged**